lib/cadet_web/admin_views/admin_assessments_view.ex

```diff
@@ -66,6 +66,23 @@ defmodule CadetWeb.AdminAssessmentsView do
     )
   end
 
+  def render("leaderboard.json", %{leaderboard: leaderboard}) do
+    render_many(leaderboard, CadetWeb.AdminAssessmentsView, "contestEntries.json",
+      as: :contestEntries
+    )
+  end
+
+  def render("contestEntries.json", %{contestEntries: contestEntries}) do
+    transform_map_for_view(
+      contestEntries,
+      %{
+        student_name: :student_name,
+        answer: & &1.answer["code"],
+        final_score: "final_score"
+      }
+    )
+  end
+
```

Comment:
The template name should be `contestEntry.json`, because it renders a single contest entry (which you are rendering multiple of inside `leaderboard.json`.

Likewise, the parameter name.